<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1662323187683' stop='1662323195034'>
    <name>TC001</name>
    <title>TC001</title>
    <test-cases>
        <test-case start='1662323188757' status='pending' stop='1662323188758'>
            <name>Login positive _ATASK001</name>
            <title>Login positive _ATASK001</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace>undefined</stack-trace>
            </failure>
        </test-case>
        <test-case start='1662323188759' status='passed' stop='1662323195034'>
            <name>Login again</name>
            <title>Login again</title>
            <labels>
                <label name='story' value='ATASK101'/>
            </labels>
            <parameters/>
            <steps>
                <step start='1662323188948' status='undefined'>
                    <name>enter username</name>
                    <title>enter username</title>
                    <attachments>
                        <attachment title='username' source='7c14d7b4-f7f4-48b5-b289-b9efb8fa73cd-attachment.png' type='image/png' size='27259'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <description type='text'>Login into LetCode</description>
        </test-case>
        <test-case start='1662323195034' status='pending' stop='1662323195034'>
            <name>Login New</name>
            <title>Login New</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Temporarily disabled with xit</message>
                <stack-trace>undefined</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>